squad/ci/backend/lava.py

```diff
@@ -205,10 +206,27 @@ def __download_full_log__(self, job_id):
         response = requests.get(url, params=payload)
         return response.content
 
-    def __get_job_logs__(self, job_id):
-        logger.debug("Retrieving logs job: %s" % job_id)
-        log_data = self.__download_full_log__(job_id)
-        return self.__parse_log__(log_data)
+    def __download_test_log__(self, raw_log, log_start, log_end):
+        return_lines = ""
+        if not log_start:
+            return return_lines
+        log_start_line = int(log_start)
+        log_end_line = None
+        if log_end:
+            log_end_line = int(log_end)
+        else:
+            log_end_line = log_start_line + 2  # LAVA sometimes misses the signals
```

Comment:
Do you think it's nice to check whether `log_start_line <= log_end_line`? I was just thinking of it now, but most likely it's not needed.

Reply:
I didn't see such case in LAVA. End always at least 1 greater than Start. I saw cases where lava missed the start or end by a few lines.